application/app.py

```diff
@@ -334,7 +334,7 @@ def aus_temp_rainfall():
       'year': fire['Year'],
       'avg_annual_temp': fire['Avg Annual Temp'],
       'avg_annual_rainfall' : fire['Avg Annual Rainfall'],
-      'temp_difference': fire['temp_differnce'],
```

Comment:
The jupyter notebook file  temp_rainfall.ipynb has "temp_differnce" . I'm totally fine with changing it, but everyone will need to repopulate that collection

Reply:
good call—I've reverted back to the original key.